libs/feature/search/src/lib/metadata-view/data-preview-map/data-preview-map.component.ts

```diff
@@ -0,0 +1,88 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core'
+import { map, startWith } from 'rxjs/operators'
+import { MdViewFacade } from '../state/mdview.facade'
+import { DatasetFinderService, LinkUsage } from '@geonetwork-ui/feature/dataviz'
+import {
+  MAP_CTX_LAYER_XYZ_FIXTURE,
+  MapContextModel,
+} from '@geonetwork-ui/feature/map'
+import { MetadataLinkValid } from '@geonetwork-ui/util/shared'
+import { fromLonLat } from 'ol/proj'
+import { BehaviorSubject, combineLatest } from 'rxjs'
+
+@Component({
+  selector: 'gn-ui-data-preview-map',
+  templateUrl: './data-preview-map.component.html',
+  styleUrls: ['./data-preview-map.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class DataPreviewMapComponent {
+  availableLinks$ = this.mdViewFacade.metadata$.pipe(
+    map((metadata) => ('links' in metadata ? metadata.links : [])),
+    map(
+      (links) =>
+        links.filter((link) => !('invalid' in link)) as MetadataLinkValid[]
+    ),
+    map((links) =>
+      links.filter(
+        (link) =>
+          this.datasetFinder.getLinkUsages(link).indexOf(LinkUsage.MAP) > -1
+      )
+    ),
+    startWith([])
+  )
+  dropdownChoices$ = this.availableLinks$.pipe(
```

Comment:
this list is not customizable.
`availableLinks` should be in a service/class which could be extended ?
Not mandatory for now.